docs/en/wallet-solution.rst

```diff
@@ -84,56 +84,44 @@ The returning Entity Configuration of the Wallet Provider MUST contain the
 attributes listed below:
 
 Header
-^^^^^^
-+---------+-----------------------------------------------------------------+
-| **Key** | **Value**                                                       |
-+---------+-----------------------------------------------------------------+
-| alg     | Algorithm used to verify the token signature (e.g., ES256).     |
-+---------+-----------------------------------------------------------------+
-| kid     | Thumbprint of the public key used for signing.                  |
-+---------+-----------------------------------------------------------------+
-| typ     | Media type, set to ``entity-statement+jwt``.                    |
-+---------+-----------------------------------------------------------------+
+^^^^^^^
+.. list-table::
+    :widths: 20 80
+    :header-rows: 1
+
+    * - **Key**
+      - **Value**
+    * - alg
+      - Algorithm used to verify the token signature. It MUST be one of the possibile values indicated in this `table <https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/algorithms.html>`_ (e.g., ES256).
+    * - kid
+      -  Thumbprint of the public key used for signing.
```

Comment:
```suggestion
      -  Thumbprint of the public key used for signing, according to [RFC 7638](https://www.rfc-editor.org/rfc/rfc7638.html]
```

please check that RFC 7638 is in the referenced standard and then use the rst ref pointing to standards.rst rfc 7638 instead of the markdown href

Reply:
Yes, RFC 7638 is present in docs/common/standards.rst. I'll edit the sentence as you requested